MODULE.bazel

```diff
@@ -0,0 +1,72 @@
+module(
+    name = "rules_kotlin",
+    compatibility_level = 1,
+    repo_name = "dev_io_bazel_rules_kotlin",
+    version = "1.6.0",
+)
+
+bazel_dep(name = "platforms", version = "0.0.4")
+bazel_dep(name = "rules_pkg", version = "0.7.0")
+bazel_dep(name = "bazel_skylib", version = "1.2.1")
+bazel_dep(name = "stardoc", repo_name = "io_bazel_stardoc", version = "0.5.3")
+bazel_dep(name = "rules_java", version = "5.1.0")
+
+kt = use_extension("//src/main/starlark/core/repositories:bzlmod_setup.bzl", "kt_bzlmod_ext")
+
+use_repo(kt, "rkt_1_7")
+
+use_repo(kt, "rkt_1_6")
+
+use_repo(kt, "rkt_1_5")
+
+use_repo(kt, "rkt_1_4")
+
+use_repo(kt, "legacy")
+
+use_repo(kt, "com_github_jetbrains_kotlin")
+
+use_repo(kt, "kt_java_stub_template")
+
+use_repo(kt, "com_github_pinterest_ktlint")
+
+bazel_dep(name = "rules_proto", version = "4.0.0")
```

Comment:
nit: why are these `bazel_dep`s down here, instead of with the group above?

Reply:
Fixed.